dev/AccessControl/GetSecurityDescriptorForAppContainerNames.cpp

```diff
@@ -0,0 +1,100 @@
+﻿#include <pch.h>
+#include <wil/resource.h>
+#include <wil/result.h>
+#include <accctrl.h>
+#include <aclapi.h>
+#include <userenv.h>
+#include "Security.AccessControl.h"
+#include "SecurityDescriptorHelpers.h"
+#include <vector>
+
+using namespace std;
+
+STDAPI GetSecurityDescriptorForAppContainerNames(
+    uint32_t accessRequestCount,
+    _In_reads_(accessRequestCount)
+    const AppContainerNameAndAccess* accessRequests,
+    _In_opt_ PSID principal,
```

Comment:
Should this also be const?

Reply:
Sadly the way `PSID` is defined is as `typedef PVOID PSID`, and we do not have an equivalent `typedef PCVOID PCSID` in the platform SDK (... yet) ... so if you wanted to add a conditional typedef for `PCSID` that's probably fine.

(The `SID` struct type becomes `PISID`, but you have to reinterpret_cast from PSID to SID*.)